Source/SwiftLintFramework/Rules/Lint/ArrayInitRule.swift

```diff
@@ -69,7 +69,7 @@ extension ArrayInitRule {
                 return
             }
 
-            violationPositions.append(node.positionAfterSkippingLeadingTrivia)
+            violationPositions.append(memberAccess.name.positionAfterSkippingLeadingTrivia)
```

Comment:
Changing the position where the violation triggers to the method made the delegating rule a bit easier to implement. Of course, this could be done differently having a separate method here for the type-safe variant. I think, the new position fits actually better, though.

Reply:
Fine by me.